### PR TITLE
Made minor changes to SDK_AutoML_Forecasting_Model_Training_Example file and moved file from community to official

### PR DESCRIPTION
**Pull request by sudarshan-SpringML**
SDK_AutoML_Forecasting_Model_Training_Example
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

**Comment by sudarshan-SpringML** [comment]
ran fine in colab and workbench

File: notebooks/official/sdk/SDK_AutoML_Forecasting_Model_Training_Example.ipynb [hunk 1]
```diff
@@ -0,0 +1,941 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Forecasting_Model_Training_Example.ipynb:1]: <h2>Create a Managed Time Series Dataset from BigQuery  -&gt; Create a Vertex Time Series Dataset from a Big Query table</h2><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Forecasting_Model_Training_Example.ipynb:1]: <p> </p><p> </p><h2>Launch a Training Job to Create a Model -&gt; Run the training job</h2><p> </p><p>Once we have defined your training script, we will create a model. -&gt; Next, run the training job to train the model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Forecasting_Model_Training_Example.ipynb:1]: <h2>Fetch Model Evaluation Metrics -&gt; Get the model evaluation metrics</h2><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Forecasting_Model_Training_Example.ipynb:1]: <h1>Run Batch Prediction -&gt; Make a batch prediction</h1><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Forecasting_Model_Training_Example.ipynb:1]: <h3>Create Output BigQuery Dataset -&gt; Create a BigQuery dataset for the batch prediction output</h3><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Forecasting_Model_Training_Example.ipynb:1]: <h2>Visualize the Forecasts -&gt; Visualize the forecast</h2><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

**Comment by sudarshan-SpringML** [comment]
All comments resolved. Ran fine in colab and workbench

**Comment by andrewferlitsch** [comment]
There seems to be a permission problem on creating the dataset:

1 ds = aiplatform.datasets.TimeSeriesDataset.create(
Step #3:       2     display_name="iowa_liquor_sales_train", bq_source=[TRAINING_DATASET_BQ_PATH]
Step #3:       3 )
Step #3:       5 ds.resource_name
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/google/cloud/aiplatform/datasets/time_series_dataset.py:127, in TimeSeriesDataset.create(cls, display_name, gcs_source, bq_source, project, location, credentials, request_metadata, labels, encryption_spec_key_name, sync)
Step #3:     119 metadata_schema_uri = schema.dataset.metadata.time_series
Step #3:     121 datasource = _datasources.create_datasource(
Step #3:     122     metadata_schema_uri=metadata_schema_uri,
Step #3:     123     gcs_source=gcs_source,
Step #3:     124     bq_source=bq_source,
Step #3:     125 )
Step #3: --> 127 return cls._create_and_import(
Step #3:     128     api_client=api_client,
Step #3:     129     parent=initializer.global_config.common_location_path(
Step #3:     130         project=project, location=location
Step #3:     131     ),
Step #3:     132     display_name=display_name,
Step #3:     133     metadata_schema_uri=metadata_schema_uri,
Step #3:     134     datasource=datasource,
Step #3:     135     project=project or initializer.global_config.project,
Step #3:     136     location=location or initializer.global_config.location,
Step #3:     137     credentials=credentials or initializer.global_config.credentials,
Step #3:     138     request_metadata=request_metadata,
Step #3:     139     labels=labels,
Step #3:     140     encryption_spec=initializer.global_config.get_encryption_spec(
Step #3:     141         encryption_spec_key_name=encryption_spec_key_name
Step #3:     142     ),
Step #3:     143     sync=sync,
Step #3:     144 )
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/google/cloud/aiplatform/base.py:750, in optional_sync.<locals>.optional_run_in_thread.<locals>.wrapper(*args, **kwargs)
Step #3:     748     if self:
Step #3:     749         VertexAiResourceNounWithFutureManager.wait(self)
Step #3: --> 750     return method(*args, **kwargs)
Step #3:     752 # callbacks to call within the Future (in same Thread)
Step #3:     753 internal_callbacks = []
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/google/cloud/aiplatform/datasets/dataset.py:334, in _Dataset._create_and_import(cls, api_client, parent, display_name, metadata_schema_uri, datasource, project, location, credentials, request_metadata, labels, encryption_spec, sync, create_request_timeout, import_request_timeout)
Step #3:     251 @classmethod
Step #3:     252 @base.optional_sync()
Step #3:     253 def _create_and_import(
Step #3:    (...)
Step #3:     268     import_request_timeout: Optional[float] = None,
Step #3:     269 ) -> "_Dataset":
Step #3:     270     """Creates a new dataset and optionally imports data into dataset when
Step #3:     271     source and import_schema_uri are passed.
Step #3:     272 
Step #3:    (...)
Step #3:     331             Instantiated representation of the managed dataset resource.
Step #3:     332     """
Step #3: --> 334     create_dataset_lro = cls._create(
Step #3:     335         api_client=api_client,
Step #3:     336         parent=parent,
Step #3:     337         display_name=display_name,
Step #3:     338         metadata_schema_uri=metadata_schema_uri,
Step #3:     339         datasource=datasource,
Step #3:     340         request_metadata=request_metadata,
Step #3:     341         labels=labels,
Step #3:     342         encryption_spec=encryption_spec,
Step #3:     343         create_request_timeout=create_request_timeout,
Step #3:     344     )
Step #3:     346     _LOGGER.log_create_with_lro(cls, create_dataset_lro)
Step #3:     348     created_dataset = create_dataset_lro.result()
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/google/cloud/aiplatform/datasets/dataset.py:459, in _Dataset._create(cls, api_client, parent, display_name, metadata_schema_uri, datasource, request_metadata, labels, encryption_spec, create_request_timeout)
Step #3:     403 """Creates a new managed dataset by directly calling API client.
Step #3:     404 
Step #3:     405 Args:
Step #3:    (...)
Step #3:     448         An object representing a long-running operation.
Step #3:     449 """
Step #3:     451 gapic_dataset = gca_dataset.Dataset(
Step #3:     452     display_name=display_name,
Step #3:     453     metadata_schema_uri=metadata_schema_uri,
Step #3:    (...)
Step #3:     456     encryption_spec=encryption_spec,
Step #3:     457 )
Step #3: --> 459 return api_client.create_dataset(
Step #3:     460     parent=parent,
Step #3:     461     dataset=gapic_dataset,
Step #3:     462     metadata=request_metadata,
Step #3:     463     timeout=create_request_timeout,
Step #3:     464 )
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/google/cloud/aiplatform_v1/services/dataset_service/client.py:625, in DatasetServiceClient.create_dataset(self, request, parent, dataset, retry, timeout, metadata)
Step #3:     620 metadata = tuple(metadata) + (
Step #3:     621     gapic_v1.routing_header.to_grpc_metadata((("parent", request.parent),)),
Step #3:     622 )
Step #3:     624 # Send the request.
Step #3: --> 625 response = rpc(
Step #3:     626     request,
Step #3:     627     retry=retry,
Step #3:     628     timeout=timeout,
Step #3:     629     metadata=metadata,
Step #3:     630 )
Step #3:     632 # Wrap the response in an operation future.
Step #3:     633 response = gac_operation.from_gapic(
Step #3:     634     response,
Step #3:     635     self._transport.operations_client,
Step #3:     636     gca_dataset.Dataset,
Step #3:     637     metadata_type=dataset_service.CreateDatasetOperationMetadata,
Step #3:     638 )
Step #3: 
Step #3: File ~/.local/lib/python3.9/site-packages/google/api_core/gapic_v1/method.py:154, in _GapicCallable.__call__(self, timeout, retry, *args, **kwargs)
Step #3:     151     metadata.extend(self._metadata)
Step #3:     152     kwargs["metadata"] = metadata
Step #3: --> 154 return wrapped_func(*args, **kwargs)
Step #3: 
Step #3: File ~/.local/lib/python3.9/site-packages/google/api_core/grpc_helpers.py:59, in _wrap_unary_errors.<locals>.error_remapped_callable(*args, **kwargs)
Step #3:      57     return callable_(*args, **kwargs)
Step #3:      58 except grpc.RpcError as exc:
Step #3: ---> 59     raise exceptions.from_grpc_error(exc) from exc
Step #3: 
Step #3: PermissionDenied: 403 Permission 'aiplatform.datasets.create' denied on resource '//aiplatform.googleapis.com/projects/vertex-ai-dev/locations/us-central1' (or it may not exist). [reason: "IAM_PERMISSION_DENIED"
Step #3: domain: "aiplatform.googleapis.com"
Step #3: metadata {
Step #3:   key: "permission"
Step #3:   value: "aiplatform.datasets.create"
Step #3: }
Step #3: metadata {
Step #3:   key: "resource"
Step #3:   value: "projects/vertex-ai-dev/locations/us-central1"
Step #3: }
Step #3: ]

**Comment by sudarshan-SpringML** [comment]
Error resolved

**Review by andrewferlitsch — APPROVED** [review]

**Review by ivanmkc — CHANGES_REQUESTED** [review]
We already have a forecasting notebook so we don't need another./

**Comment by ivanmkc** [comment]
Closing.

**Pull request closed without merging**